modules/local/adata/merge/templates/merge.py

```diff
@@ -26,12 +26,12 @@ def format_yaml_like(data: dict, indent: int = 0) -> str:
             yaml_str += f"{spaces}{key}: {value}\\n"
     return yaml_str
 
-adatas = dict(zip("${names}".split(), [sc.read_h5ad(f) for f in "${h5ads}".split()]))
+adatas = dict(zip("${names}"[1:-1].split(", "), [sc.read_h5ad(f) for f in "${h5ads}".split()]))
```

Comment:
Ah I missed that values are inserted differently than paths. But if we remove `index_unique`, we don't need it anyways:
```suggestion
adatas = [sc.read_h5ad(f) for f in "${h5ads}".split()]
```

Reply:
This also makes a change necessary in the next line, as the data structure changed